mlflow/metrics/genai/prompts/v1.py

```diff
@@ -17,33 +17,50 @@
 }
 grading_system_prompt_template = PromptTemplate(
     """
-Please act as an impartial judge and evaluate the quality of the provided output which
-attempts to produce output for the provided input based on a provided information.
-
-You'll be given a grading format below which you'll call for each provided information,
-input and provided output to submit your justification and score to compute the {name} of
-the output.
-
-Input:
+====
+Task
+====
+You are an impartial judge. You will be given an input that was sent to a machine
+learning model, and you will be given an output that the model produced.
+There is additional information that was provided to the LLM to generate the output.
+
+Your task is to determine a numerical score called {name} based on the input and output.
+A definition of {name} and a grading rubric are provided below.
+You must use the grading rubric to determine your score. You must also justify your score.
+
+Examples are included below for reference. You must read them and make sure you
+understand before completing the task.
+
+=====
+Input
+=====
 {input}
 
-Provided output:
+======
+Output
+======
 {output}
 
+======================
+Additional information
+======================
 {grading_context_columns}
 
-Metric definition:
+=================
+Metric definition
+=================
 {definition}
 
-Below is your grading criteria:
+==============
+Grading rubric
+==============
 {grading_prompt}
 
 {examples}
 
-And you'll need to submit your grading for the {name} of the output,
-using the following in json format:
-Score: [your score number for the {name} of the output]
-Justification: [your step by step reasoning about the {name} of the output]
+You must output a JSON object with the following fields:
```

Comment:
Do you know how reliable is it for the judge to always output json? I am wondering if it's simpler and more reliable to let it output the justification and score in separate lines instead, and it will also be easier to reuse by other LLMs who can't follow complex instruction that well

Reply:
Seems to be very reliable